### DISCUSSION TITLE
Hardware questions

### DISCUSSION THREAD

**Discussion by geraldFS** [philibertc/micronova_controller#19] [Q&A] [answered]
![IMG_20211030_132059](https://user-images.githubusercontent.com/35431341/139585574-a2421313-5308-4322-99e7-2c3d643b10d7.jpg)
Hi there,

I have an oven with the Micronova Mod0047 Evo control.

This has a D-Sub connector to the outside, 18V / 5V Serial / GND (source k3a.me)
And the 4-pin serial interface (CN9) is soldered onto the circuit board.

Does jamend know whether I can also pick up the signals with your circuit board layout at the D-Sub output?

Or should the internal connection be used?

Regards
Gerald

**Comment by philibertc** [comment]
Hello
According to k3a.me here is the pinout of the DB9 connector. However, the fixed 5V is missing on its schematic, and it is necessary to have it. I would recommend you to use the internal connector (less pins, so much less possible errors), but you can try to find the fixed 5V on the DB9 connector (it should be easy to find, since we know the GND pin).
![image](https://user-images.githubusercontent.com/57588282/139661953-bf31de9b-1ead-49c7-88ae-2328a50148dc.png)

**Comment by geraldFS** [comment, marked as answer]
Hi there,

I once measured the 9-pin D-SUB socket and here is the pin assignment.
Maybe someone still needs them ;-)

I use an **ITC Layma** with a Micronova mod0047 EVO control-board.

Gerald 

```
PIN DB9 
Pin 7 & 6   | GND
Pin 1       | 5V
Pin 3       | 20V
Pin 8       | Serial
```
 

![D-S9](https://user-images.githubusercontent.com/35431341/140316087-724e40ad-915c-4860-b91a-9b5031fe01eb.png)

**Reply by philibertc** [reply]
Hello, thank you for your schematics! I will soon create a markdown page with all the information that may be useful, like this diagram.
